.github/workflows/test-yjit.yaml

```diff
@@ -11,10 +11,16 @@ jobs:
           - '3.2'
           - '3.3'
           # ADD NEW RUBIES HERE
-    name: Test (${{ matrix.os }}, ${{ matrix.ruby }})
+        rubyopt:
+          - '--yjit'
+          - '--yjit --yjit-stats=quiet'
+        exclude:
+          - ruby: '3.2'
+            rubyopt: '--yjit --yjit-stats=quiet'
```

Comment:
Why is this configuration excluded?

Reply:
`--yjit-stats` exists on ruby 3.2, but it dumps all the stat info after the interpreter exits, and it doesn't include the new metric added (`ratio_in_yjit`, only in 3.3+), so having that for 3.2 isn't valuable here.

`--yjit-stats=quiet` was introduced in 3.3, and _doesn't_ dump that info to stdout.

I can put up a PR with a comment for clarity.

Reply:
for your consideration: https://github.com/DataDog/dd-trace-rb/pull/4010